### PR TITLE
Add action EDIT_TEMPLATE

### PR DESCRIPTION
**Pull request by EmelineLeduc**
**Detailed purpose of the PR**
This PR adds action to EDIT_TEMPLATE.
-> https://trello.com/c/h2YQ3Ufr/2717-app-review-ajouter-action-pour-edittemplate

**Result and observation**
Kapture_multiple-template

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Comment by EmelineLeduc** [comment]
[[app-review] Ajouter action pour EDIT_TEMPLATE](https://trello.com/c/h2YQ3Ufr/2717-app-review-ajouter-action-pour-edittemplate)

File: packages/@coorpacademy-app-review/src/actions/ui/answers.ts [hunk 1]
```diff
@@ -29,25 +29,27 @@ export type EditAnswerAction = {
...
   switch (questionType) {
     case 'qcm':
     case 'qcmGraphic':
     case 'qcmDrag': {
-      return includes(newValue, userAnswers)
-        ? pull(newValue, userAnswers)
-        : flatten([...userAnswers, newValue]);
+      return includes(newValue[0], userAnswers)
+        ? pull(newValue[0], userAnswers)
+        : flatten([...userAnswers, newValue[0]]);
```
Comment (esa-coorp) [inline packages/@coorpacademy-app-review/src/actions/ui/answers.ts:39]:
```suggestion
      const firstValue = head(newValue);
      return includes(firstValue, userAnswers)
        ? pull(firstValue, userAnswers)
        : flatten([...userAnswers, firstValue]);
```

File: packages/@coorpacademy-app-review/src/actions/ui/answers.ts [hunk 2]
```diff
@@ -29,25 +29,27 @@ export type EditAnswerAction = {
...
     }
     case 'basic':
     case 'slider':
-      return [newValue];
+      return newValue;
+    case 'template':
+      return newValue;
```
Comment (esa-coorp) [inline packages/@coorpacademy-app-review/src/actions/ui/answers.ts:45]:
```suggestion
    case 'template':
      return newValue;
```

Comment (esa-coorp) [reply packages/@coorpacademy-app-review/src/actions/ui/answers.ts:45]: le case `basic`, `slider`, et `template` retourne la même chose

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Review by StefanoA1 — APPROVED** [review]

**Review by jomaora — APPROVED** [review]

**Pull request merged**